Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -264,13 +264,7 @@ static HRESULT __DWriteTextLayoutCreate(CFAttributedStringRef string, CFRange ra
     // Used to separate runs for attributes which DWrite does not handle until drawing (e.g. Foreground Color)
     uint32_t incompatibleAttributeFlag = 0;
     CFRange attributeRange;
-
-    // Find the range of the first set of attributes and skip it, since the underlying DWriteTextFormat has already internalized it
```

Comment:
Yea, I thought we would be fine skipping the first one but I forgot about kerning, my b